### ISSUE TITLE
What does this mean

### ISSUE THREAD

**Issue by MOAboAli** [google/tirg#11]
https://github.com/google/tirg/blob/4d9fb23511e34bc180cb7551e4414eb2f938fc84/img_text_composition_models.py#L85

labels = torch.tensor(range(x.shape[0])).long()

is this mean labels is always zero ?
